app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+  def show
+  end
+end
```

Comment:
If this controller is never being used we can delete it too.  Looks like the `HomesController` is being used in `routes.rb` to display the empty div with an id of app.